node/narwhal/tests/common/primary.rs

```diff
@@ -209,6 +209,28 @@ impl TestNetwork {
         sleep(Duration::from_millis(MAX_BATCH_DELAY * 2)).await;
         self.validators.values().all(|v| v.primary.current_round() <= halt_round)
     }
+
+    // Checks if the committee is coherent in storage for all nodes (not quorum) over a range of
+    // rounds.
+    pub fn is_committee_coherent<T>(&self, rounds_range: T) -> bool
+    where
+        T: RangeBounds<u64> + IntoIterator<Item = u64>,
+    {
+        rounds_range.into_iter().all(|round| {
+            self.validators.values().map(|v| v.primary.storage().get_committee(round).unwrap()).dedup().count() == 1
```

Comment:
are the entries consecutive, so that `dedup` will work?

Reply:
Currently yes, committees are static—we may need to tweak this once dynamic committees are in.